schema/metadata_pointset_point.json

```diff
@@ -14,16 +14,27 @@
       "type": "boolean"
     },
     "baseline_value": {
+      "definition": "Used for telemetry validation in a lab environment. Represents the expected baseline point value",
```

Comment:
these are not strictly "in a lab environment" -- since the baseline could be compared against in other environments. Really it's just the "expected baseline value of the point" -- but can't say much about the environment where it would be used.